SimPEG/electromagnetics/natural_source/simulation.py

```diff
@@ -317,7 +390,7 @@ def fields(self, m=None):
 ###################################
 # 3D problems
 ###################################
-class Simulation3DPrimarySecondary(BaseNSEMSimulation):
+class Simulation3DPrimarySecondary(BaseFDEMSimulation):

```

Comment:
@JKutt: I reverted this just for now to make sure that the tests still pass with a light re-naming, but I think this is a good call- we are not too far off. I will start (or revive) an issue where we can chat through the necessary changes